mkdocs.yml

```diff
@@ -19,6 +19,7 @@ nav:
 
   - Autonomous Economic Agent Framework:
     - Welcome: 'index.md'
+    - Agent-oriented approach: 'agent-oriented-approach.md'
```

Comment:
Call it `Intro to agent-oriented software development` ?

Reply:
Please link to this section from the welcome section as well.

Reply:
It's not really an introduction. Called it agent-oriented development.